README.md

```diff
@@ -135,6 +122,19 @@ CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\vcvars64.
 python x.py build
 ```
 
+#### Specifying an ABI
+
+Each specific ABI can also be used from either environment (for example, using
+the GNU ABI in powershell) by using an explicit build triple. The available
+Windows build triples are:
+- `x86_64-pc-windows-gnu` - The GNU ABI (using GCC)
+- `x86_64-pc-windows-msvc` - The MSVC ABI
```

Comment:
What about i686? I mean granted nobody should be using i686 but there are people that might want it.